portfoliostylesheet.css

```diff
@@ -0,0 +1,332 @@
+* {
+	box-sizing:border-box;
+}
+
+a {
+    text-decoration: none;
+}
+a:link, a:visited {
+    color: #B4535A;
+}
+
+a:hover {
+    color: #D0D2CA;
+}
+
+html { 
+	font-size:16px;
+	font-family:"verdana", sans-serif;
+}
+
+body {
+	margin:0px;
+	padding:0px;
+}
+	
+h3 {
+	font-family:"avenir book";
+	text-size:1rem;
+	color:#B28537;
+}
+
+.splitGrid {
+	display: grid;
+	grid-template-columns: 4fr 1fr ;
+	margin-left: 15%;
+	margin-right:15%;
+}
+
+
+.footer {
+	margin-left:40%;
```

Comment:
Use `margin: 0 auto` to center it instead

Reply:
this didn't work, I think we talked about it the other day as being a confusing thing.